app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -0,0 +1,18 @@
+name: Intervention
+description: An area that contains personalised content to the user
+body: |
+  The intervention is used to show personalised content. For instance, if the user has visited multiple
+  pages in the same area of the site, we might want let them know that there are other pages on GOV.UK
+  that would be useful to them. This component would be used to add this personalised content and would
+  indicate to the user that this is not normally part of the page, but has been added for them specifically.
+
+  Right now the contents of the component are static, as the MVP of personalised content is only for Start a Business.
+  Since many pages will use this component with the same text, we hard-code it here for now.
+accessibility_criteria: |
```

Comment:
The component also contains a link, so you should add the shared accessibility criteria for links, add the following (as in https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/docs/accordion.yml):

```
shared_accessibility_criteria:
  - link
```